src/uwtools/tests/drivers/test_jedi.py

```diff
@@ -0,0 +1,203 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+JEDI driver tests.
+"""
+import datetime as dt
+import logging
+from pathlib import Path
+from unittest.mock import DEFAULT as D
+from unittest.mock import Mock, call, patch
+
+import yaml
+from iotaa import asset, external
+from pytest import fixture
+
+from uwtools.config.formats.yaml import YAMLConfig
+from uwtools.drivers import jedi
+from uwtools.scheduler import Slurm
+from uwtools.tests.support import regex_logged
+
+# Fixtures
+
+
+@fixture
+def cycle():
+    return dt.datetime(2024, 2, 1, 18)
+
+
+# Driver fixtures
+
+
+@fixture
+def config(tmp_path):
+    return {
+        "jedi": {
+            "execution": {
+                "batchargs": {
+                    "export": "NONE",
+                    "nodes": 1,
+                    "stdout": "/path/to/file",
+                    "walltime": "00:02:00",
+                },
+                "envcmds": [
+                    "module load some-module",
+                    "module load jedi-module",
+                ],
+                "executable": "/path/to/qg_forecast.x",
+                "mpiargs": ["--export=ALL", "--ntasks $SLURM_CPUS_ON_NODE"],
+                "mpicmd": "srun",
+            },
+            "configuration_file": {
+                "base_file": str(tmp_path / "config.yaml"),
```

Comment:
I am sorry to do this...we've gone round and round about the name of this file, but I'm just noticing one other potential issue. The base file is named `config.yaml` as is the fixture on line 71. Both are in the same place. I'm not sure what sort of debugging issue this may present in the future, but it might be good to go ahead and fix that ambiguity now.

I suggest changing this instance to`base.yaml`.